scripts/notify-suppliers-of-awarded-briefs.py

```diff
@@ -1,75 +1,48 @@
 #!/usr/bin/env python3
 """
-If a brief has been awarded suppliers need to be notified. This
-script notifies suppliers of all awarded briefs in the previous day by default.
-Alternatively, when supplied with a list of BriefResponse IDs, it will notify just the suppliers for those responses.
+If a brief has been awarded suppliers need to be notified. This script notifies
+suppliers of all awarded briefs in the previous day by default.
 
-Usage:
-    notify-suppliers-of-awarded-briefs.py <stage> <govuk_notify_api_key> <govuk_notify_template_id>
-        [options]
-
-Options:
-    -h, --help                                      Display this screen
-    -v, --verbose                                   Verbosity level
-    --dry-run                                       Log without sending emails
-    --awarded_at=<awarded_at>                       Notify applicants to briefs awarded on this date, defaults to
-                                                    # yesterday (date format: YYYY-MM-DD)
-    --brief_response_ids=<brief_response_ids>       List of brief response IDs to send to (for resending failures)
+Alternatively, when supplied with a list of BriefResponse IDs, it will notify
+just the suppliers for those responses.
+"""
 
+__examples__ = """
 Examples:
-    ./scripts/notify-suppliers-of-awarded-briefs.py preview myToken notifyToken t3mp1at3id --awarded_at=2017-10-27
-    ./scripts/notify-suppliers-of-awarded-briefs.py preview myToken notifyToken t3mp1at3id --dry-run --verbose
-
+    ./scripts/notify-suppliers-of-awarded-briefs.py \\
```

Comment:
Is the double-slash here intentional?
When I try and run these examples I am just getting the usage prompt e.g.
```
$ ./scripts/notify-suppliers-of-awarded-briefs.py --stage=preview --notify-api-token=notifyToken --notify-template-id=t3mp1at3id --awarded_at=2017-10-27
Usage:
    notify-suppliers-of-awarded-briefs.py <stage> <govuk_notify_api_key> <govuk_notify_template_id>
        [options]
```
Sorry if I'm missing something obvious?

Reply:
The double-slash is so that Python doesn't treat it as escaping the newline character, so that when it is printed the newline is present.

The error you're getting is very weird... Are you running the examples with this PR checked out?? 🤔

Reply:
Ah yes, I think I was confused by the force push, it improved once I reset!

Reply:
Ah, I found a better way to get around the escaping, I've made it a raw-string.